README.md

```diff
@@ -40,6 +40,7 @@ Workflow engine using [WDL](https://github.com/broadinstitute/wdl/blob/wdl2/SPEC
   * [GET /workflows/:version/:id/outputs/:call](#get-workflowsversionidoutputscall)
   * [GET /workflows/:version/:id/logs/:call](#get-workflowsversionidlogscall)
   * [GET /workflows/:version/:id/logs](#get-workflowsversionidlogs)
+  * [GET /workflows/:version/:id/call-metadata](#get-workflowsversionidcallmetadata)
```

Comment:
Since this whole thing is unlikely to be completely done this sprint, I'll point out the hot of the press ticket for next sprint DSDEEPB-1264 where this whole thing will become a workflow-level affair. IMO we might as well have the endpoint named in a way that we won't change next week.